eox_nelp/middleware.py

```diff
@@ -5,6 +5,8 @@
 classes:
     ExtendedProfileFieldsMiddleware: Set extended_profile_fields in registration form.
```

Comment:
could you please add the new middleware here

Reply:
[dbfda6b](https://github.com/eduNEXT/eox-nelp/pull/127/commits/dbfda6b358edbc876e43c300976cf5585023b8f8)